.rubocop.yml

```diff
@@ -15,6 +16,7 @@ AllCops:
     - '**/*.gemspec'
   DisplayCopNames: true
   RunRailsCops: true
+  TargetRubyVersion: 2.4
```

Comment:
I figured I'd throw this in here for new projects, but I don't know if we really need it.